include/Framework/RandomNumberSeedService.h

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file RandomNumberSeedService.h
+ * @brief Conditions object for random number seeds
+ * @author Jeremy Mans, University of Minnesota
+ */
+
+#ifndef FRAMEWORK_RANDOMNUMBERSEEDSERVICE_H_
+#define FRAMEWORK_RANDOMNUMBERSEEDSERVICE_H_
+
+/*~~~~~~~~~~~~~~~*/
+/*   Framework   */
+/*~~~~~~~~~~~~~~~*/
+#include "Framework/ConditionsObject.h"
+#include "Framework/ConditionsObjectProvider.h"
+
+
+namespace ldmx {
+
+/** @class RandomNumberSeedService
+ * System for consistent seeding of random number generators
+ * 
+ * The system can be configured in a number of different ways.  In general, seeds are constructed based on 
+ * a master seed.  That master seed can be constructed in a number of ways, chosen in the python configuration.
+ *  (a) The master seed can be the run number of the first run observed by the service ("Run")
+ *  (b) The master seed can be based on the current time ("Time")
+ *  (c) The master seed can be provided in the python configuration ("External")
+ * 
+ * Individual seeds are then constructed using the master seed and a simple hash based on the name of the seed.
+ * Seeds can also be specified in the python file, in which case no autoseeding will be performed.
+ */
+class RandomNumberSeedService : public ConditionsObject, public ConditionsObjectProvider {
+ public:
+  /** Conditions object name */
+  static constexpr const char* CONDITIONS_OBJECT_NAME="RandomNumberSeedService";
```

Comment:
`constexpr` implies constant so the `const` keyword is not needed here.  Also, why use a char* instead of a string?